src/app/icd/ICDManager.h

```diff
@@ -128,6 +129,7 @@ class ICDManager : public ICDListener
     static void OnTransitionToIdle(System::Layer * aLayer, void * appState);
 
     static uint8_t OpenExchangeContextCount;
+    static uint8_t CheckInRequestCount;
```

Comment:
```suggestion
   uint8_t mOpenExchangeContextCount;
   uint8_t mCheckInRequestCount;
```

These don't need to be static anymore.

Reply:
Pretty sure @jmartinez-silabs had a reason to put that static.

Reply:
Yes my bad, When updating from ICDEventManager mechanic to the ICDNotifer, the static functions became just public (non-static), I forgot to move this static variable to the object too